examples/list-proposals/index.js

```diff
@@ -0,0 +1,19 @@
+require('dotenv').config();
+
+const { AdminClient } = require('@openzeppelin/defender-admin-client');
+
+async function main() {
+	const creds = { apiKey: process.env.ADMIN_API_KEY, apiSecret: process.env.ADMIN_API_SECRET };
+	const client = new AdminClient(creds);
+
+	const proposals = await client.listProposals({
+		limit: 10,
+		next: undefined,
```

Comment:
Should we document how "next" can be used as well?

Reply:
Yep, good point